docs/ManualClusterSetup.md

```diff
@@ -4,14 +4,17 @@ This document describes requirements and best practices for setting up a cluster
 
 ## Requirements
 
-2 servers (bare metal or VMs) with a modern 64-bit Linux OS, and Layer-2 network (Ethernet) connectivity between them.
+2 servers (bare metal or VMs) with a modern 64-bit Linux OS, each with a unique hostname.
 
 They must have the following software installed.
  * Docker v1.8 or greater: [Experimental Docker](https://experimental.docker.com)
  * etcd installed and available on each node: [etcd Documentation](https://coreos.com/etcd/docs/2.0.8/)
+ * Consul installed on one node [Consul Documentation](https://www.consul.io/docs/index.html)
+   * Consul is required for Docker Networking.  etcd support is planned, but currently broken.
+   * Do not run Consul in a Docker container; it must be available when the Docker daemon starts.
  * `ipset`, `iptables`, and `ip6tables` kernel modules.
 
-## Best Practices
+## Getting Docker 1.8 Experimental Binaries
```

Comment:
We should be fully supported on CoreOS now. The plugin path thing is no longer a problem. So remove the section recommending powerstrip

And the experimental binaries are never updated by docker. So current plan of record is that the docker binary is attached to the releases page. And that's the version they should use.
(Also see this https://github.com/Metaswitch/calico-docker/wiki/Current-recommended-Experimental-Docker-version-for-testing-Calico)